src/MemoryTracing/include/MemoryTracing/ProcessMemoryInfoProducer.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef MEMORY_TRACING_PROCESS_MEMORY_INFO_PRODUCER_H_
+#define MEMORY_TRACING_PROCESS_MEMORY_INFO_PRODUCER_H_
+
+#include "GrpcProtos/Constants.h"
+#include "MemoryTracing/MemoryInfoProducer.h"
+
+namespace orbit_memory_tracing {
+
+// This class periodically produces the ProcessMemoryUsage information retrieved from
```

Comment:
```suggestion
// This class periodically produces the `ProcessMemoryUsage` information retrieved from
```

Reply:
Done (now its at a different location than before, but I changed it accordingly). Thanks!